src/blockchain/web3_extentions/lido_contracts.py

```diff
@@ -54,6 +56,22 @@ def _load_contracts(self):
         self._load_staking_router()
         self._load_dsm()
 
+        if variables.SIMPLE_DVT_STAKING_STRATEGY_MELLOW_CONTRACT is not None:
+            self.simple_dvt_staking_strategy: SimpleDVTStakingStrategyContract = cast(
+                SimpleDVTStakingStrategyContract,
+                self.w3.eth.contract(
+                    address=variables.SIMPLE_DVT_STAKING_STRATEGY_MELLOW_CONTRACT,
+                    ContractFactoryClass=SimpleDVTStakingStrategyContract,
+                ),
+            )
+            self.erc20: ERC20Contract = cast(
```

Comment:
Rename `erc20` -> 'weth'

Reply:
I removed the contract from here, now it is loaded as property from other contracts as you proposed